modules/@apostrophecms/area/ui/apos/components/AposAreaEditor.vue

```diff
@@ -267,10 +285,26 @@ export default {
       this.hoveredWidget = _id;
       this.hoveredNonForeignWidget = nonForeignId;
     },
-    updateWidgetFocused(widgetId) {
-      this.focusedWidget = widgetId;
+    updateWidgetFocused({ _id, focus }) {
```

Comment:
```suggestion
    updateWidgetFocused({ _id, scrollIntoView = false }) {
```

I'd rather call it `scrollIntoView` than `focus`.